test/integration/helpers/integration-test-editor.js

```diff
@@ -100,14 +103,19 @@ export function Editor( { testBlocks, settings = {} } ) {
  * @param {Object | Array} testBlocks    Block or array of block settings for blocks to be tested.
  * @param {boolean}        useCoreBlocks Defaults to true. If false, core blocks will not be registered.
  * @param {Object}         settings      Any additional editor settings to be passed to the editor.
+ * @param {Array}          skippedBlocks Blocks that should be skipped during registration.
  */
 export async function initializeEditor(
 	testBlocks,
 	useCoreBlocks = true,
-	settings
+	settings,
+	skippedBlocks = []
 ) {
 	if ( useCoreBlocks ) {
-		registerCoreBlocks();
+		const coreBlocks = __experimentalGetCoreBlocks().filter(
+			( { name } ) => ! skippedBlocks.includes( name )
+		);
+		registerCoreBlocks( coreBlocks );
```

Comment:
@glendaviesnz, I couldn't think of the best fix at the moment.

I think integration tests initialize blocks more than once without proper editor state cleanup.

Reply:
the blocks were being [cleaned up after each text here](https://github.com/WordPress/gutenberg/pull/52823/files#diff-62fabf61141260f50291d0966b5c6104311191a232f1ad98da86579538420b31R75), bit the footnotes block also adds a `core/footnote` format, and the formats were not being cleaned up - I have pushed an update which also removes any core formats that have been added - seems to fix the issue.

Reply:
Thank you, @glendaviesnz!